src/components/document-read-only-page/document-infobar.tsx

```diff
@@ -3,66 +3,38 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-
-import type { DateTime } from 'luxon'
 import React from 'react'
 import { Trans, useTranslation } from 'react-i18next'
 import { InternalLink } from '../common/links/internal-link'
 import { ShowIf } from '../common/show-if/show-if'
-import {
-  DocumentInfoLineWithTimeMode,
-  DocumentInfoTimeLine
-} from '../editor-page/document-bar/document-info/document-info-time-line'
 import styles from './document-infobar.module.scss'
-import { useCustomizeAssetsUrl } from '../../hooks/common/use-customize-assets-url'
-
-export interface DocumentInfobarProps {
-  changedAuthor: string
-  changedTime: DateTime
-  createdAuthor: string
-  createdTime: DateTime
-  editable: boolean
-  noteId: string
-  viewCount: number
-}
+import { useApplicationState } from '../../hooks/common/use-application-state'
+import { NoteInfoLineCreated } from '../editor-page/document-bar/note-info/note-info-line-created'
+import { NoteInfoLineUpdated } from '../editor-page/document-bar/note-info/note-info-line-updated'
 
-export const DocumentInfobar: React.FC<DocumentInfobarProps> = ({
-  changedAuthor,
-  changedTime,
-  createdAuthor,
-  createdTime,
-  editable,
-  noteId,
-  viewCount
-}) => {
+/**
+ * Renders an infobar with metadata about the current note.
+ */
+export const DocumentInfobar: React.FC = () => {
   const { t } = useTranslation()
-  const assetsBaseUrl = useCustomizeAssetsUrl()
+  const noteDetails = useApplicationState((state) => state.noteDetails)
 
+  // TODO Check permissions ("writability") of note and show edit link depending on that.
```

Comment:
?

Reply:
The pencil icon (link to editor) should only be shown when the current user has write permissions for the note.
I'm not exactly sure how to get that information as either the note data is returned from the server or we receive a forbidden response.